lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -63,6 +63,7 @@ def __init__(self, fig, rect=None, *args, **kwargs):
           *elev*             Elevation viewing angle (default 30)
           *zscale*           [%(scale)s]
           *sharez*           Other axes to share z-limits with
+          *persp*            Perspective projection (default True)
```

Comment:
I think it would be better to have a `proj` argument instead, which (at the moment) could take either `'persp` or `ortho` values, and default to `persp`.